tests/lib/installers.py

```diff
@@ -377,3 +377,151 @@ def install_dovecot(device):
         device.sendline('apt-get install dovecot-imapd dovecot-pop3d -y')
         device.expect(['Processing triggers for dovecot-core'], timeout=90)
         device.expect(device.prompt)
+
+def install_ovpn_server(device, remove=False, _user='lan', _ip="ipv4"):
+    '''Un/Install the OpenVPN server via a handy script'''
+    device.sendline('cd')
+    device.expect_exact('cd')
+    device.expect(device.prompt)
+    device.sendline('ls -l /etc/init.d/openvpn')
+    index = device.expect(['(\\s{1,}\\d{4}()\\s{1,}\\/etc\\/init\\.d\\/openvpn)'] + device.prompt, timeout=90)
+
+    # do we want to remove it?
+    if remove:
+        if index == 0:
+            device.expect(device.prompt)
+            # be brutal, the server may not be responding to a stop
+            device.sendline('killall -9 openvpn')
+            device.expect(device.prompt, timeout=60)
+            device.sendline('./openvpn-install.sh')
+            device.expect('Select an option.*: ')
+            device.sendline('3')
+            device.expect_exact('Do you really want to remove OpenVPN? [y/n]: n')
+            device.sendcontrol('h')
+            device.sendline('y')
+            device.expect(device.prompt, timeout=90)
+        return
+
+    # do the install
+    if index != 0:
+        dev_ip = device.get_interface_ipaddr(device.iface_dut)
+        device.sendline('apt-get update')
+        device.expect(device.prompt)
+        device.sendline('curl -O https://raw.githubusercontent.com/Angristan/openvpn-install/master/openvpn-install.sh')
+        device.expect(device.prompt, timeout=90)
```

Comment:
We can't use the local version?

Reply:
yes we can, i just did not integrate it....yet....

Reply:
Sure, give it some consideration. Esp. for increasing reliability of anything automated.